README.md

```diff
@@ -67,9 +72,30 @@ _Note: binaries available via `cargo install` is coming soon._
 
 ## Getting started
 
-We recommend to browse the [examples](https://github.com/rs-ipfs/rust-ipfs/tree/master/examples) and
-[tests](https://github.com/rs-ipfs/rust-ipfs/tree/master/tests) in order to see how to use Rust-IPFS
-in different scenarios.
+We recommend browsing the [examples](https://github.com/rs-ipfs/rust-ipfs/tree/master/examples), the [http crate tutorial](https://github.com/rs-ipfs/rust-ipfs/tree/master/http#getting-started) and [tests](https://github.com/rs-ipfs/rust-ipfs/tree/master/tests) in order to see how to use Rust-IPFS in different scenarios.
+
+### Running the tests
+
+The project currently features unit, integration, conformance and interoperability tests. Unit and integation tests can be run with:
+
+```bash
+$ cargo test
```

Comment:
I'd be biased on including `--ẁorkspace` at least here, otherwise it might be quite surprising that no tests from ipfs-{http,unixfs} get executed.

Not sure if this needs to be on the interop tests.

Reply:
> Not sure if this needs to be on the interop tests.

Not strictly necessary (just filters out extra unixfs and http tests)